FAQ page for the documentation (#948)
adap/flower: doc/source/faq-section.rst

FAQ
===

> Find here the answers of questions frequently asked. 

Reviewer: ```suggestion
This page collects answers to commonly asked questions about Federated Learning with Flower.
```

.. dropdown:: :fa:`eye,mr-1` Can I combine federated learning with blockchain?

>     Yes, of course. A list of available examples using Flower within a blockchain environment are available here:

Reviewer: ```suggestion
    Yes, of course. A list of available examples using Flower within a blockchain environment is available here:
```

    * `Flower meets Nevermined GitHub Repository <https://github.com/nevermined-io/fl-demo/tree/master/image-classification-flower>`_.
    * `Flower meets Nevermined YouTube video <https://www.youtube.com/watch?v=A0A9hSlPhKI>`_.
    * `Flower meets KOSMoS <https://www.kosmos-bmbf.de/wp-content/uploads/sites/13/2021/05/Talk-Flower-Summit-2021.pdf>`_.
    * `Flower meets Talan blog post <https://www.linkedin.com/pulse/federated-learning-same-mask-different-faces-imen-ayari/?trackingId=971oIlxLQ9%2BA9RB0IQ73XQ%3D%3D>`_ .
    * `Flower meets Talan GitHub Repository <https://gitlab.com/Talan_Innovation_Factory/food-waste-prevention>`_ .

.. dropdown:: :fa:`eye,mr-1` Does Flower support federated learning on Android devices?

>     Yes, it does. Please find have a look to our `blog post <https://flower.dev/blog/2021-12-15-federated-learning-on-android-devices-with-flower>`_ or check out the `example on Flower GitHub <https://github.com/adap/flower/tree/main/examples/android>`_.

Reviewer: ```suggestion
    Yes, it does. Please take a look at our `blog post <https://flower.dev/blog/2021-12-15-federated-learning-on-android-devices-with-flower>`_ or check out the `Android code example on GitHub <https://github.com/adap/flower/tree/main/examples/android>`_.
```

> .. dropdown:: :fa:`eye,mr-1` How can I setup Flower on a Raspberry Pi?

Reviewer: ```suggestion
.. dropdown:: :fa:`eye,mr-1` How can I run Federated Learning on a Raspberry Pi?
```
